### PR TITLE
update homepage

### PR DESCRIPTION
**Pull request by jacobkhong**
1) Added Banner

2) Created the Search bar(unresponsive)
- The search button for now directs you to ALL Houses(index)
- code are in the partial file _search, render on the home view

3) Added Featured Project
- Added @houses variable on the Pages Controller so that it can be access
- Added @top_houses to extract the top 4 houses by the price
- code are in the partial file _featured, render on the home view

### CODE REVIEW

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="navbar navbar-expand-sm navbar-light navbar-lewagon
 ">
-  <%= link_to houses_path, class: "navbar-brand" do %>
+  <%= link_to root_path, class: "navbar-brand" do %>
```
Comment (douglashjf) [inline app/views/shared/_navbar.html.erb:3]: @jacobkhong is Root_path not back to the landing page?

File: app/controllers/pages_controller.rb
[the remark this answers is not in the document]
Comment (jacobkhong) [reply app/controllers/pages_controller.rb:1]: Yes ideally what you want to change to?

[the remark this answers is not in the document]
Comment (douglashjf) [reply app/controllers/pages_controller.rb:1]: Highest # of offers is one example to show popularity, most-viewed could be too complicated for now

**Review by douglashjf — APPROVED** [review]
looking good

**Pull request merged**